docs/configure-artifact-repository.md

```diff
@@ -75,7 +75,8 @@ $ cat > policy.json <<EOF
          "Effect":"Allow",
          "Action":[
             "s3:PutObject",
-            "s3:GetObject"
+            "s3:GetObject",
+            "s3:ListBucket"
          ],
```

Comment:
"ListBucket" is required as of 3.4 (written for the Artifact Visualization feature) because we do a check to determine if the file is a directory, which requires this